### DISCUSSION TITLE
3ds max export for glb

### DISCUSSION THREAD

**Discussion by andypoly** [atteneder/glTFast#451] [Q&A] [answered]
We are trying to get glb models loading at runtime from 3ds max. So far an artist has supplied one at least 10 times too big and with no scene index (-1). This is using the built in gltf exporter.

Has anyone worked on glb export from 3ds and had any problems with scaling and scene index?

**Comment by atteneder** [comment, marked as answer]
I never used 3ds max exporter, but according to the glTF spec, the unit in use is meters. If you set up your scene in 3ds max accordingly (like, to any real-world unit like meter, inches, millimeter, yards,...) I'd expect it to convert it to meters and export properly. If not, I'd file a bug report with them.

I answered about the scene index in your other discussion thread.

hth
